WordPress/Classes/Services/SiteVerticalsService.swift

```diff
@@ -0,0 +1,22 @@
+
+typealias SiteVerticalsServiceCompletion = (Result<[SiteVertical]>) -> Void
+
+/// Abstracts obtention of site verticals
+protocol SiteVerticalsService {
+    func verticals(for: Locale, type: SiteType, completion: @escaping SiteVerticalsServiceCompletion)
+}
+
+/// Mock implementation of the SiteVerticalsService
+final class MockSiteVerticalsService: SiteVerticalsService {
+    func verticals(for: Locale = .current, type: SiteType, completion: @escaping SiteVerticalsServiceCompletion) {
+        let result = Result.success(mockVerticals())
+
+        completion(result)
+    }
+
+    private func mockVerticals() -> [SiteVertical] {
+        return [ SiteVertical(id: Idenfier(value: "SV 1"), title: "Vertical 1"),
```

Comment:
`Idenfier` appears to be misspelled. I looked to see if this was used elsewhere or if perhaps there was a compile error. Upon doing so, its looking like `SiteVerticalsService.swift` has not yet been added to `WordPress/WordPress.xcodeproj/project.pbxproj`.

Reply:
Speaking of `Idenfier`, could that be a `typealias` on `String`? I know it's still early days, so that discussion might be premature.

Reply:
I think it should be a type. I know usually strings are used as identifiers, but those, to me, are two completely different types because they represent different abstractions.

It think some people call that "primitive obsession" or "primitive envy" https://refactoring.guru/smells/primitive-obsession

And oops, I will make sure to add that class to the project!